crates/flightrepl/src/completer.rs

```diff
@@ -0,0 +1,570 @@
+use crate::EditorHelper;
+use arrow_flight::flight_service_client::FlightServiceClient;
+use datafusion::arrow::array::{Array, StringArray};
+use datafusion::sql::sqlparser::keywords::ALL_KEYWORDS;
+use rustyline::Context;
+use rustyline::completion::{Completer, Pair};
+use std::sync::Arc;
+use std::time::Duration;
+use tokio::sync::{RwLock, oneshot};
+use tokio::time::interval;
+use tonic::transport::Channel;
+
+#[derive(Debug, Clone)]
+struct StringValue {
+    #[allow(dead_code)]
+    original: Arc<str>,
```

Comment:
```suggestion
    _original: Arc<str>,
```

nit: prefixing with underscore should work?

Reply:
Done